README.md

```diff
@@ -0,0 +1,22 @@
+# Animator
+Animator is a small and very simple animation tool.
+
+![Animator screenshot](animator.png)
+
+
+Building
+--------
+
+Simply go into source directory and type ``make``.
+
+
+Copyright
+------------
+
+Original 2000 version by Jonathan Villemure, enhanced and updated for Haiku in 2018 by Arnav Bhatt.
+
+
+Contact
```

Comment:
Drop contact info.

Reply:
Done

Reply:
Enhanced and updated?  All you are doing if updating the readme.  This part can be dropped.  Your updates with be tagged with you as the author of the commit(s) you make.  No need to add it to readme unless you've made significant changes to the code.

Reply:
Oops, I forgot about this one, wait a minute.